src/components/Footer/Footer.jsx

```diff
@@ -21,7 +20,6 @@ import RenderCopyrightAndLinksSection from './RenderCopyrightAndLinksSection';
  */
 const Footer = () => {
   const theme = useTheme();
-  const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'));
   const isReallySmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
```

Comment:
It's more of a stylistic choice (so it'll probably be for a separate PR if we decide to tackle it), but I wonder if we should start switching the direction of the footer earlier at the `md` (at 900px) breakpoint instead of `sm` (at 600px) @andycwilliams 

Breakpoint at `md`:
<img src="https://github.com/codeforpdx/PASS/assets/14917816/80b7251a-e83b-41f2-a702-eba83e4d1dfe" height="300">
<img src="https://github.com/codeforpdx/PASS/assets/14917816/3ca9ed5f-589e-49e1-963a-b3dafaf68052" height="300">

Breakpoint at `sm`:
<img src="https://github.com/codeforpdx/PASS/assets/14917816/6f31e17d-1222-49b3-8a40-400d6f6e83b7" height="300">
<img src="https://github.com/codeforpdx/PASS/assets/14917816/27b6953b-e880-43b3-9370-de5d4a0d3ec4" height="300">